.vscode/launch.json

```diff
@@ -0,0 +1,53 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Python: Remote Debug",
+      "type": "python",
+      "request": "attach",
+      "connect": {
+        "host": "localhost",
+        "port": 5678
+      },
+      "pathMappings": [
+        {
+          "localRoot": "${workspaceFolder}/bakery-src/scripts/",
+          "remoteRoot": "/workspace/enki/venv/lib/python3.8/site-packages/bakery_scripts"
+        }
+      ],
+      "justMyCode": false
+    },
+    {
+      "name": "Java: Remote Debug",
+      "type": "java",
+      "projectName": "xhtml-validator",
+      "request": "attach",
+      "hostName": "localhost",
+      "port": 7777
+    },
+    {
+      "name": "Bakery JS: Remote Debug",
+      "type": "node",
+      "sourceMaps": true,
+      "address": "localhost",
+      "port": 9229,
+      "localRoot": "${workspaceFolder}/bakery-js/dist/",
+      "remoteRoot": "/workspace/enki/bakery-js/dist/",
+      "request": "attach",
+      "skipFiles": ["<node_internals>/**"]
+    },
+    {
+      "name": "Mathify: Remote Debug",
```

Comment:
I was able to debug all the JS/TS files by doing [this](https://github.com/openstax/enki/compare/sk-ce-1953...phil-ce-1953) (set a `NODE_OPTIONS` environment var and edit `.vscode/launch.json`)

Reply:
Thanks, @philschatz. I'll pull it to my branch and test it.